nilearn/decomposition/multi_pca.py

```diff
@@ -25,7 +25,8 @@ def session_pca(imgs, mask_img, parameters,
                 memory_level=0,
                 memory=Memory(cachedir=None),
                 verbose=0,
-                copy=True):
+                copy=True,
+                random_state=None):
```

Comment:
Just curious is there a scikit-learn convention that dictates whether random_state should be set to None or a fixed integer by default?